renpy/compat/pickle.py

```diff
@@ -104,3 +108,182 @@ def dump(o, f, highest=False):
 
     def dumps(o, highest=False):
         return pickle.dumps(o, pickle.HIGHEST_PROTOCOL if highest else PROTOCOL)
+
+    # The python AST module changed significantly between python 2 and 3. Old-style
+    # screenlang support records raw python ast nodes into the rpyc data, making these
+    # impossible to load normally. This function can be called for node name `name`
+    # It will return None if the node type does not require any changes, or return
+    # a wrapper class with a custom __setstate__ implementation that will cause the pickle
+    # machinery to do the right thing. There are some things that cannot be handled by this
+    # (as the type of the node to be emitted is not fixed at that point), so those
+    # are handled by a NodeTransformer in the ast.Module replacement.
+    # Note: this isn't a complete python 2 -> python 3 ast conversion, we just convert
+    # what is needed to still support old-style screens (Ren'py 6.17 and below)
+    def ast_rewrite(name):
```

Comment:
This function is obsolete, as it stands.

Reply:
I've removed it.